mf.py

```diff
@@ -1,9 +1,15 @@
+import pycuda.autoinit
+import pycuda.gpuarray as gpuarray
+import skcuda.linalg as linalg
 import numpy as np
+import math
+import skcuda.misc as misc
+
 
 
 class MF():
     
-    def __init__(self, R, K, alpha, beta, iterations):
+    def __init__(self, gpu, R, K, alpha, beta, iterations):

```

Comment:
I think gpu as an option can come at last, or at least behind R, and K. Also would be more clear to name it `use_gpu`

```
def __init__(self, R, K, alpha, beta, iterations, use_gpu):
```